frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BasePullBranchFastForwardOnlyAction.java

```diff
@@ -99,6 +100,9 @@ public void actionPerformed(AnActionEvent anActionEvent) {
       }
 
       if (FetchUpToDateTimeoutStatus.isUpToDate(gitRepository)) {
+        IntelliJNotificationCompat.notifyInfo(project,
+            getString("action.GitMachete.BasePullBranchFastForwardOnlyAction.task-title"),
+            "No new fetch has been performed since the latest fetch happened less than a minute ago");
```

Comment:
Hardcoding `a minute` is questionable... note that we've got a constant for that, `com.virtuslab.gitmachete.frontend.actions.common.FetchUpToDateTimeoutStatus#FETCH_ALL_UP_TO_DATE_TIMEOUT_MILLIS`... maybe define another constant next to  `FETCH_ALL_UP_TO_DATE_TIMEOUT_MILLIS`, with a human-readable string like `"a minute"`